src/Microsoft.AspNet.Mvc.Core/ApplicationModels/IApiExploringModel.cs

```diff
@@ -0,0 +1,7 @@
+﻿namespace Microsoft.AspNet.Mvc.ApplicationModels
+{
+    public interface IApiExploringModel
```

Comment:
Yes, I did try to avoid confusion... but ended up having some inconsistencies instead.